packages/std/src/math/int128.rs

```diff
@@ -519,14 +515,14 @@ mod tests {
 
     #[test]
     fn int128_new_works() {
-        let num = Int128::new([1; 16]);
+        let num = Int128::from_be_bytes([1; 16]);
         let a: [u8; 16] = num.to_be_bytes();
         assert_eq!(a, [1; 16]);
 
         let be_bytes = [
             0u8, 222u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 0u8, 1u8, 2u8, 3u8,
         ];
-        let num = Int128::new(be_bytes);
+        let num = Int128::from_be_bytes(be_bytes);
```

Comment:
We now test `from_be_bytes ` in `int128_new_works `. Could you add one test function for each of the two constructors?